buildtools/scripts/DeployArtifacts.groovy

```diff
@@ -64,6 +64,8 @@ private OptionAccessor parseOptions(arguments)
 def deployJars()
 {
     deployToRemote(ceRepoUrl, ceRepoId, 'org.mule', 'mule', version, 'pom')
+    deployToRemote(ceRepoUrl, ceRepoId, 'org.mule', 'mule-extensions', version, 'pom')
+    deployToRemote(ceRepoUrl, ceRepoId, 'org.mule.modules', 'mule-module-validation', version, 'jar')
```

Comment:
No, because although module-module-validation groupId is org.mule.modules its Maven module is a submodule of extensions, not modules. Then it is not listed in the modules pom, nor in the extensions pom. I've done this until we have a definition about how we will handle this in Mule 4.